app/src/main/java/io/apicurio/registry/rest/ArtifactsResource.java

```diff
@@ -3,10 +3,13 @@
 import io.apicurio.registry.rest.beans.ArtifactMetaData;
 import io.apicurio.registry.rest.beans.EditableMetaData;
 import io.apicurio.registry.rest.beans.Rule;
+import io.apicurio.registry.rest.beans.RuleType;
```

Comment:
This could also go into types package?

Reply:
Well this is interesting.  Nice catch.  I'm not sure how this happened - must be a generator bug.  I'll look into it.